Documentation/UsingQtCreator.md

```diff
@@ -0,0 +1,20 @@
+## Using Qt Creator for working with SerenityOS
+
+You can follow the video tutorial [on YouTube](https://www.youtube.com/watch?v=NcTEaRB_cak), or the instructions below.
```

Comment:
That video is a bit outdated, so I think it might be better to just write everything properly here.